NEWS

```diff
@@ -1,3 +1,41 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new program, ignition-validate, for validating Ignition configs
+    - Add "overwrite" field to "files", "directories", and "links" sections for
+      deleting preexisting items at the node's path
+    - Add "options" field to "raid" section for specifying arbitrary mdadm
+      options
+    - Add "append" field to "files" section for appending to preexisting files
+    - Add support for specifying custom certificate authorities to use when
```

Comment:
s/custom/additional/?  We still accept the default CAs IIUC.

Reply:
A snippet from https://golang.org/pkg/crypto/tls/: 
```
// RootCAs defines the set of root certificate authorities
// that clients use when verifying server certificates.
// If RootCAs is nil, TLS uses the host's root CA set.
RootCAs *x509.CertPool
```

Based on this I believe when a user sets a custom CA, the default CAs are not used.

However now that I'm double checking things, the documentation Crawford wrote for the new CAs field is: "the list of additional certificate authorities (in addition to the system authorities) to be used for TLS verification when fetching over `https`".

I'm betting our docs are wrong for this field, based on the documentation of the field Ignition sets when given custom CAs. I'll investigate further to see which is correct.

Reply:
Crawford and I discussed this a couple weeks ago and concluded that the behavior documented in the config spec is the desired one.  (It's less brittle when sourcing data from multiple servers, and we can always add a flag later if anyone wants the more paranoid approach.)

Perhaps we should have a test case for this.

Reply:
Oh, and after reading the code a little more closely, Ignition does load in the system cert pool before parsing additional CAs, so the documentation on the field is correct.

As a test this config runs successfully: https://gist.github.com/dgonyeo/0f424941139700f5a978b3c914e5803f

I'll update the news file.

Reply:
It looks like I can set the environment variable `SSL_CERT_FILE` to get go to use a non-default cert file for the system pool, so it should be a reasonable test to write. I'll add it, although I don't think we should block this release on it since I just tested this behavior manually.